Revit_Core_Engine/Convert/ToRevit.cs

```diff
@@ -38,41 +40,77 @@ public static partial class Convert
         /****              Public methods               ****/
         /***************************************************/
 
+        [Description("Converts BH.oM.Geometry.SettingOut.Grid to a Revit Grid or MultiSegmentGrid.")]
+        [Input("grid", "BH.oM.Geometry.SettingOut.Grid to be converted.")]
+        [Input("document", "Revit document, in which the output of the convert will be created.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
+        [Input("refObjects", "Optional, a collection of objects already processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("grid", "Revit Grid or MultiSegmentGrid resulting from converting the input BH.oM.Geometry.SettingOut.Grid.")]
```

Comment:
Same comments as above in this entire file. Many outputs don't match the output type.